service/src/main/java/uk/gov/hmcts/reform/fpl/model/RemovedPlacement.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import lombok.Builder;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Value
+@Builder(toBuilder = true)
+@Jacksonized
+public class RemovedPlacement {
+    String removalReason;
+    Placement removedPlacement;
```

Comment:
Just renaming to placement.

Reply:
https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=4211&id=FPL%3Aservice&open=AYa9H-kM33bV0uSTuojU